src/psbtv2.ts

```diff
@@ -298,6 +298,28 @@ export abstract class PsbtV2Maps {
 
     return bw.render().toString(format);
   }
+
+  public copy(to: PsbtV2) {
```

Comment:
Okay. Glad this abstract class has some usefulness. I think this copy method is fine, but two things:
- Why can't serialize work? The constructor is essentially copying maps from the serialized psbt. The intent of what I described to you out of band was that the serialized psbt was an api boundary between different polymorphs so that valid PSBTs could be ensured at constructor and `serialize`.
- In other words: Copying maps bypasses the validation defined in the constructor, so it could create a psbtv2 in  an invalid psbt status. 

Request: This should be indicated in comments/jsdoc that it has potential to copy maps outside of this kind of control.

Reply:
Added a new warning comment: [008fb32](https://github.com/Shadouts/unchained-bitcoin/pull/1/commits/008fb32d299b7906b38690198b97743f82ef4a31)

For those following along at home, the reason `copy` is required is for compatibility with the ledger api that needs a copy method (see relevant code in ledger's vendor code in uc-wallets [here](https://github.com/unchained-capital/unchained-wallets/blob/ledger-v2/src/vendor/ledger-bitcoin/src/appClient.ts#L200))